cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/ApplicationMeta.java

```diff
@@ -28,10 +30,19 @@ public class ApplicationMeta {
 
   private final String id;
   private final ApplicationSpecification spec;
+  private final Long created;
+  @Nullable
+  private final String author;
+  @Nullable
+  private final String description;
```

Comment:
Discussed this offline as well, so the problem is that the if we add `ChangeSummaryResponse` object here, it might not make sense here during deployment (we pass ApplicationMeta as a parameter as discussed before): https://github.com/cdapio/cdap/blob/develop/cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/deploy/pipeline/ApplicationRegistrationStage.java#L59

because we intend to write to the table and the object is a response object - so it isn't good for readability right?

Reply:
Also resonating this comment: https://github.com/cdapio/cdap/pull/14493/files#r986079104
Why do we need these 3 fields here if we have them has individual columns?